libs/pages/environments/src/lib/ui/page-settings/page-settings.tsx

```diff
@@ -0,0 +1,25 @@
+import { Button } from '@console/shared/ui'
+import { useParams } from 'react-router'
+
+/* eslint-disable-next-line */
+export interface PageSettingsProps {}
```

Comment:
Let's completely remove the props from page since a page will never receive any prop